### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by AndrewTal** [tomer196/GUDI#1]
Hello, I followed the following process for training:

1. Download the COMPAS dataset and extract it to obtain two training files: COMPAS-1x.csv and pahs-cata-34072-xyz.
2. Modify the training data path: data/aromatic_datapaths.py
    elif args.dataset == "cata":
    csv_path = "XXXXX/COMPAS-1x.csv"
    xyz_path = "XXXXX/pahs-cata-34072-xyz"
3. Modify the configuration file args_edm.py
    If the following parameters are not added, the verification process will directly report an error.
    self.add_argument (“--orientation”, type=bool, default=True)

After training for 5000 epochs, the generated results are basically empty, and the loss basically does not decrease, remaining around 0.112. Is there anything else we need to pay attention to?

**Comment by tomer196** [comment]
Hi,
The loss you describe is exactly what you should converge to.
The 'args. orientation' was deprecated during the research, and now it's inferred from the dataset type. 
You shouldn't use it (and anyway for the catacondensed data it should be False). 
 I saw now one place it was still in code. In the visualization scripts, This is the reason you got wrong visualization. I will push a fix for that shortly.

**Comment by AndrewTal** [comment]
Thank you for your reply! We can reproduce the results in the paper stably. Thanks again!
